kong/db/strategies/cassandra/connector.lua

```diff
@@ -9,14 +9,15 @@ CassandraConnector.__index = CassandraConnector
 
 
 local function wait_for_schema_consensus(self)
-  if not self.connection then
+  local conn = self.super.get_stored_connection(self)
```

Comment:
the Cassandra strategy does `self.super.get_stored_connection`; the Postgres strategy simply does `self:get_stored_connection`. AFAICT they seem functionally equivalent. Is the explicit `super` (and additional metatable logic) necessary?

Reply:
(No strong feeling on this — I'm more asking whether there was a technical reason or if it was a stylistic choice, though consistency between strategies would be nice if the difference is unnecessary)

Reply:
Got it, I see it now